dropwizard-app/src/main/java/org/coner/boundary/EventBoundary.java

```diff
@@ -9,7 +9,7 @@
  */
 public class EventBoundary extends AbstractBoundary<org.coner.api.entity.Event, Event, org.coner.hibernate.entity.Event> {
 
-    static EventBoundary instance;
+    private static EventBoundary instance;
```

Comment:
I almost fixed this while I was commenting, but I had a hunch I might cause a merge conflict. Thanks for handling it